frontend/src/pages/MediaPage.js

```diff
@@ -18,6 +19,8 @@ const MediaPage = ({ mediaItem, classes }) => (
       <ProfileButton variant="flat" color="inherit" />
     </AppBar>
     <div className={ classes.body }>
+      <MotdBanner />
```

Comment:
I think this is a shibboleth that we should refactor this into a general "Page" component.